templates/AWS/ApiGatewayAuthorizer.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
`template` usually refers to SAM templates. Can you instead call the folder `sample-events` or `sample-inputs` or `input-events` or something that sounds better?

Reply:
done